health_monitor/src/health_monitor.cpp

```diff
@@ -108,7 +108,7 @@ void HealthMonitor::handle_diagnostics(const diagnostic_msgs::DiagnosticArrayPtr
             (msg->status[i].values.size() > 0) &&
             (msg->status[i].values[0].key == "Code"))
         {
-            errorValues |= stoi(msg->status[i].values[0].value);
+            errorValues |= stol(msg->status[i].values[0].value);
```

Comment:
@patapufate just to be on the safe side, use `stoll` instead.

Reply:
I was going to say the same. I tested with stoll. If you'd prefer I can test again with stol

Reply:
As you and @hidmic have said, I think it will be better if we use stoll instead.